Bio/PDB/Vector.py

```diff
@@ -107,12 +111,12 @@ def rotaxis2m(theta, vector):
     rot[2, 2] = t * z * z + c
     return rot
 
-
 rotaxis = rotaxis2m
 
 
 def refmat(p, q):
-    """Return a (left multiplying) matrix that mirrors p onto q.
+    """
```

Comment:
The first line of the docstring should be on the triple-quote line, see PEP257 https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings